src/ng/directive/ngModel.js

```diff
@@ -127,8 +127,9 @@ var ngModelMinErr = minErr('ngModel');
  * };
  * ```
  *
- * @property {Array.<Function>} $viewChangeListeners Array of functions to execute whenever the
- *     view value has changed. It is called with no arguments, and its return value is ignored.
+ * @property {Array.<Function>} $viewChangeListeners Array of functions to execute whenever
+ *     a change to the view value has caused a change to the model value.
```

Comment:
Thanks for the PR. Can you please change "view value" and "model value" to "$viewValue" and "$modelValue" and link to them?